pkg/apply/applier.go

```diff
@@ -300,7 +305,8 @@ func (a *Applier) readAndPrepareObjects() (*ResourceObjects, error) {
 		}
 	}
 
-	inventoryObject, err := prune.CreateInventoryObj(invs[0], resources)
+	inv := a.InventoryFactoryFunc(invs[0])
```

Comment:
So doesn't the `splitInfos` function also need to adjust so it can find the correct inventory objects?

Reply:
No. `splitInfos` will return the objects with the `inventory-id` label, regardless of the resource type. `splitInfos` returns the inventory template as a `*resource.Info`